prow/github/client.go

```diff
@@ -663,3 +664,27 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// GetContents uses github repo contents API to retrieve the content of a file.
```

Comment:
Comment, function name, and the log in the first line of the function, all differ.